src/Model/Profile.php

```diff
@@ -100,11 +108,15 @@ public static function load(App $a, $nickname, $profile = 0, $profiledata = [],
 			return;
 		}
 
-		if (empty($a->page['aside'])) {
+		// Not found in page array?
+		if (!isset($a->page['aside'])) {
```

Comment:
Why had you replaced the ```empty()``` with ```!isset()```?

Reply:
The whole `if()` block says: If `$a->page['foo']` is empty, set it to empty string. Which makes no sense. I just changed it to `!isset()` to "say" this: If `$a->page['foo']` doesn't exist, at least set it to empty string.

Reply:
Ah okay.